src/styles/blocks/product_card.css

```diff
@@ -0,0 +1,92 @@
+.product-card {
+  font-family: "Roboto", sans-serif;
+  display: flex;
+  flex-direction: column;
+  align-items: center;
+  width: 200px;
+  background-color: var(--color-white);
+  border: 1px solid var(--border-color);
+  border-radius: 5px;
+  padding: 32px 16px 16px;
+}
+
+.product-card__product-image {
+  display: block;
+  width: 160px;
+  height: 134px;
+  margin-bottom: 40px;
+}
+
+.product-card__info {
+  margin-bottom: 16px;
+}
+
+.product-card__title {
+  font-weight: 500;
+  font-size: 12px;
+  line-height: 18px;
+  margin-bottom: 4px;
+  color: var(--main-text-color);
+}
+
+.product-card__code {
+  display: block;
+  font-weight: 400;
+  font-size: 10px;
+  color: var(--inscription-text-color);
+}
+
+.product-card__review {
+  display: flex;
+  justify-content: space-between;
+  align-items: center;
+  width: 100%;
+  margin-bottom: 24px;
+}
+
+.product-card__number-of-reviews {
+  font-weight: 400;
+  font-size: 10px;
+  color: var(--main-text-color);
+}
+
+.product-card__price-info {
+  display: flex;
+  justify-content: space-between;
+  align-items: center;
+  width: 100%;
+  margin-bottom: 16px;
+}
+
+.product-card__price-lettering {
+  font-weight: 400;
+  font-size: 12px;
+  color: var(--inscription-text-color);
+}
+
+.product-card__price {
+  font-weight: 700;
+  font-size: 16px;
+  color: var(--main-text-color);
+}
+
+.product-card__buy {
+  display: block;
+  width: 166px;
+  height: 40px;
+  line-height: 40px;
+  border-radius: 5px;
+  background-color: var(--color-blue);
+  font-weight: 700;
+  font-size: 14px;
+  text-transform: uppercase;
+  text-decoration: none;
+  text-align: center;
+  color: var(--color-white);
+}
+
+.product-card__buy:hover {
+  background-color: var(--color-white);
+  color: var(--color-blue);
+  border: 1px solid var(--color-blue);
```

Comment:
8th item of [checklist](https://github.com/mate-academy/layout_product-cards/blob/master/checklist.md)